### PR TITLE
Feature/add PriorityclassName

### PR DESCRIPTION
**Pull request by nujragan93**
Hey there 👋

This PR adds priorityClassName to dragonfly pods, addressing https://github.com/dragonflydb/dragonfly-operator/issues/132

Please let me know if I'm missing something.

### CODE REVIEW

File: api/v1alpha1/dragonfly_types.go [hunk 1]
```diff
@@ -87,6 +87,11 @@ type DragonflySpec struct {
 	// +kubebuilder:validation:Optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
 
+	// (Optional) Dragonfly pod service account name
```
Comment (Abhra303) [inline api/v1alpha1/dragonfly_types.go]: please update the comment :)

Comment (nujragan93) [reply api/v1alpha1/dragonfly_types.go]: fixed

**Comment by Abhra303** [comment]
please update manifests. Run `make generate-manifests` and push the changes. Also make sure your commit is [signed]( https://github.com/dragonflydb/dragonfly/blob/main/CONTRIBUTING.md).

**Comment by nujragan93** [comment]
> please update manifests. Run `make generate-manifests` and push the changes. Also make sure your commit is [signed](https://github.com/dragonflydb/dragonfly/blob/main/CONTRIBUTING.md).

done

File: api/v1alpha1/dragonfly_types.go [hunk 2]
```diff
@@ -92,6 +92,11 @@ type DragonflySpec struct {
 	// +kubebuilder:validation:Optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
 
+	// (Optional) Dragonfly pod priority class name
+	// +optional
+	// +kubebuilder:validation:Optional
+	PriorityClassName string `json:"PriorityClassName,omitempty"`
```
Comment (Abhra303) [inline api/v1alpha1/dragonfly_types.go]: nit: `json:"priorityClassName,`

Comment (nujragan93) [reply api/v1alpha1/dragonfly_types.go]: done

**Review by Abhra303 — APPROVED** [review]
LGTM

**Pull request merged**